src/main/java/org/folio/entlinks/service/authority/AuthoritySourceFileService.java

```diff
@@ -78,6 +84,11 @@ public AuthoritySourceFile create(AuthoritySourceFile entity) {
     return repository.save(entity);
   }
 
+  @Transactional(propagation = Propagation.REQUIRES_NEW)
+  @Retryable(
+      retryFor = OptimisticLockingException.class,
+      maxAttempts = 2,
+      backoff = @Backoff(delay = 500))
   public AuthoritySourceFile update(UUID id, AuthoritySourceFile modified) {
```

Comment:
Now, through patch we are also updating some fields (name, type, baseUrl) for ```folio``` source files. So, I think logic for propagation needs to be adjusted as well so that not to convert folio source files into local ones.

And, as @psmagin mentioned, I think we can add a validation for doing patch (and delete) through central tenant only